package.json

```diff
@@ -61,16 +62,20 @@
     "eslint-plugin-unicorn": "^47.0.0",
     "fast-json-stable-stringify": "~2.1.0",
     "fdir": "~6.0.1",
+    "fs-extra": "11.1.1",
```

Comment:
I think this is only used for `fs.readJson()`. Can you get rid of it and just use plain fs and `JSON.parse()` like other existing code in BCD?

Reply:
As part of this I'm going to make reading the json files serial like what is done in `index.ts`. `update` at the moment reads the json in parallel. Since it is as of this time 1455 json files, some systems may run in EMFILE or ENFILE errors opening that many files at one time.

Reply:
Makes sense, I guess @queengooborg and I never ran into that.

Reply:
@mzgoddard the fs-extra dependency is still here. Can it be removed now?